### PR TITLE
My first pull request. 

### PR DESCRIPTION
**Pull request by AnnaLozovytska**
I completed the first Git and GitHub Intro task. I did my first pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,24 +92,34 @@ Try it! It’s fun, we promise :)
...
 # Good luck!
+
 ---
+
 ## Hi kottans, my name is Andrii.
+
 I'm late, but I hope, I'll can study with you!
 All the best!
-___
+
```
Comment (suchov) [inline README.md:120]: Too many changes +299 −175 adding/removing for couple lines change ;)

File: README.md [hunk 2]
```diff
@@ -205,20 +258,24 @@ Bonsoir les Chats
...
-На видео курсах первой и второй недели __Coursera__, а так же на практических занятиях сайта [https://learngitbranching.js.org/](https://learngitbranching.js.org/) я узнал много интересного:
+---
+
+Было очень интересно изучать **Git** и **GitHub**.
+
+На видео курсах первой и второй недели **Coursera**, а так же на практических занятиях сайта [https://learngitbranching.js.org/](https://learngitbranching.js.org/) я узнал много интересного:
 
-* сейчас для меня все в новинку. Сам __Git__ - является для меня чем-то новым,неизведанным. Но все же, допустим команда __git init__, команда с помощью которой я научился создавать репозиторий на своем компьютере
-* удивила команда __git cherry-pick__
-* __git init, git clone, git status__ - их, и многие другие я буду точно использовать в будущем
```
Comment (suchov) [inline README.md:221]: Looks like you need to config your IDE.

**Review by suchov — CHANGES_REQUESTED** [review]

**Comment by suchov** [comment]
@AnnaLozovytska it feels like you struggle a bit. You need to check what changes you are suggesting to do with your "merge request" https://github.com/kottans/mock-repo/pull/1164/files - here is a good place to see them.
You can find things that can help you at https://discord.gg/XqNT2mGJ

**Comment by AnnaLozovytska** [comment]
I already understood what the mistake was. I opened the file with VScode and it adds the lines. I'm fixing it now. thank you

**Review by suchov — APPROVED** [review]

**Comment by suchov** [comment]
Really nice, great work @AnnaLozovytska !

**Pull request merged**